Makefile

```diff
@@ -2,5 +2,11 @@ serve:
 	docker run -it -p 4000:4000 --rm --mount type=bind,source="$(PWD)",target=/srv/jekyll jekyll/jekyll:latest /bin/bash
 	# docker run --mount type=bind,source="$(pwd)",target=/srv/jekyll/ --rm -it -p 4000:4000 -env JEKYLL_ENV=production jekyll/jekyll:latest /bin/bash
 
+serve-locally:
```

Comment:
Thanks, I got real lazy with this one so i appreciate you making this much nicer